mlir/lib/Dialect/Linalg/TransformOps/LinalgTransformOps.cpp

```diff
@@ -485,8 +485,8 @@ transform::FuseOp::apply(transform::TransformRewriter &rewriter,
       tileSizes.size() - llvm::count(tileSizes, 0), transformResults,
       [&](TilingInterface tilingInterfaceOp)
           -> FailureOr<scf::SCFTileAndFuseResult> {
-        return tileConsumerAndFuseProducerGreedilyUsingSCFForOp(
-            rewriter, tilingInterfaceOp, tileAndFuseOptions);
+        return tileConsumerAndFuseProducersUsingSCF(rewriter, tilingInterfaceOp,
```

Comment:
nit: is the 'UsingSCF' part of the name still load-bearing now that we don't distinguish between `scf.for` and `scf.forall` anymore ?

Reply:
Well, good question. This is in the SCF dialect... So I wanted to leave the "SCF" part. I dont have a strong preference, but this is living in SCF dialect.